README.md

```diff
@@ -62,16 +64,22 @@ Fully working examples of how to use these components are in `/example-crates`:
 [`electrum-client`]: https://docs.rs/electrum-client/
 [`bdk_chain`]: https://docs.rs/bdk-chain/
 
+## Dependencies
+
+- `bitcoind_rpc` depends on `bitcoind` being installed and available in `PATH` or in the `BITCOIND_EXEC` ENV variable.
+- `esplora` depends on [Blockstream's version of `electrs`](https://github.com/Blockstream/electrs)
+   being installed and available in `PATH` or in the `ELECTRS_EXEC` ENV variable.
+
 ## Minimum Supported Rust Version (MSRV)
+
 This library should compile with any combination of features with Rust 1.63.0.
 
 To build with the MSRV you will need to pin dependencies as follows:
 
 ```shell
-cargo update -p zstd-sys --precise "2.0.8+zstd.1.5.5"
-cargo update -p time --precise "0.3.20"
 cargo update -p home --precise "0.5.5"
 cargo update -p proptest --precise "1.2.0"
+cargo update -p bumpalo --precise "3.14.0" 
```

Comment:
Another rebase all green now.
@notmandatory check the new `bumpalo` CI pin.
We might need to fix that in the near future (or maybe now) ir regular CI.

Reply:
Looks like this is just a dependency for WASM and I don't think it's OK to only run those tests with `stable` rust.